app/src/main/java/data/RecipeFileHandler.java

```diff
@@ -11,4 +14,22 @@ public RecipeFileHandler(String filePath) {
         this.filePath = filePath;
     }
 
+    // 設問1: 一覧表示機能
+    public List<String> readRecipes() {
+        try {
+
+        } catch (IOException e) {
+
+        }
+        return null;
+    }
```

Comment:
レビューしながら考えなおしたのですが、
処理があっても評価に影響しないので、catchブロック内の処理はスケルトンに含めて良いかなと思いました。

```suggestion
    /**
     * 設問1: 一覧表示機能
     * recipes.txtからレシピデータを読み込み、それをリスト形式で返します。 <br> 
     * IOExceptionが発生したときは<i>Error reading file: 例外のメッセージ</i>とコンソールに表示します。
     *
     * @return レシピデータ
     */
    public List<String> readRecipes() {
        try {

        } catch (IOException e) {
            System.out.println("Error reading file:" + e.getMessage());
        }
        return null;
    }
```